### DISCUSSION TITLE
Plex Downloads Still Suck

### DISCUSSION THREAD

**Discussion by giscus** [TsekNet/website#17] [Announcements]
# Plex Downloads Still Suck

The unsatisfying story of how I’ve resolved to download media (movies, etc.) from my NAS to my phone.

https://tseknet.com/blog/plex-downloads

**Comment by LordFransie** [comment]
I ended ip just using Infuse. Its a paid solution but it fixes all the issues with downloading videos from any kind of share or from Jellyfin/Plex. 

None of these projects are particularly motivated to make downloads even functional.
